File: app/src/main/java/com/shindra/Schedule/ScheduleItem.java
package com.shindra.Schedule;

public class ScheduleItem {

    // Attributes
    private String mNextDepartureTime;
    private String mTramStationName;
    private String mTramLineLetter;

    // Constructor
    public ScheduleItem(String tramLineLetter, String tramStationName, String nextDepartureTime) {
        mNextDepartureTime = nextDepartureTime;
        mTramStationName = tramStationName;
        mTramLineLetter = tramLineLetter;
    }

    // Methods
    public String GetNextDepartureTime() { return mNextDepartureTime; }
    public String GetTramStationName() { return mTramStationName; }
    public String GetTramLineLetter() {return mTramLineLetter;}
}

File: app/src/main/java/com/shindra/Schedule/ScheduleItem.kt
package com.shindra.Schedule

class ScheduleItem(private val mNextDepartureTime: String, private val mTramStationName: String,  private val mTramLineLetter: String) {

    // Methods
    fun getNextDepartureTime(): String { return mNextDepartureTime }
    fun getTramStationName(): String { return mTramStationName }
    fun getTramLineLetter(): String { return mTramLineLetter }
}